vip_spec.xsd

```diff
@@ -259,7 +259,7 @@ reference elements for polling location
       <xs:element name="PostElectionStatus" type="CandidatePostElectionStatus" minOccurs="0" />
       <xs:element name="PreElectionStatus" type="CandidatePreElectionStatus" minOccurs="0" />
       <!-- NOTE: Still trying to get feedback or an example of what this means and how it's used -->
-      <xs:element name="SequenceOrder" type="xs:integer" />
+      <xs:element name="SequenceOrder" type="xs:integer" minOccurs="0" />
```

Comment:
As mentioned in discussion on the PR that introduced ballot style IDs to precinct, I think we can safely scratch this altogether, but that's probably a topic for a separate PR.